default/python/AI/EnumsAI.py

```diff
@@ -18,13 +13,6 @@ class PriorityType(IntEnum):
     PRODUCTION_INVASION = 9
     PRODUCTION_MILITARY = 10
     PRODUCTION_BUILDINGS = 11
-    RESEARCH_LEARNING = 12
-    RESEARCH_GROWTH = 13
-    RESEARCH_PRODUCTION = 14
-    RESEARCH_CONSTRUCTION = 15
-    RESEARCH_ECONOMICS = 16
-    RESEARCH_SHIPS = 17
-    RESEARCH_DEFENSE = 18
```

Comment:
I think I got problems loading old safe files after removing values here.

Reply:
They are supposed to be unused, so they should not be in the saves.  Will take a look.

Reply:
I have 1 old save from August 2021 it works for me both with master and this branch.